components/octavia/values.yaml

```diff
@@ -0,0 +1,135 @@
+---
+
+
+labels:
+  # (nicholas.kuechler) TODO: remove this label override when
+  # https://review.opendev.org/c/openstack/openstack-helm/+/952722
+  # is merged in
+  health_manager:
+    node_selector_key: openstack-network-node
+    node_selector_value: enabled
+
+# typically overridden by environmental
+# values, but should include all endpoints
+# required by this chart
+endpoints:
+  oslo_messaging:
+    statefulset:
+      replicas: 3
+      name: rabbitmq-server
+    hosts:
+      default: rabbitmq-nodes
+  load_balancer:
+    port:
+      api:
+        public: 443
+    scheme:
+      public: https
+    host_fqdn_override:
+      public:
+        tls:
+          secretName: octavia-tls-public
+          issuerRef:
+            name: understack-cluster-issuer
+            kind: ClusterIssuer
+
+network:
+  # configure OpenStack Helm to use Undercloud's ingress
+  # instead of expecting the ingress controller provided
+  # by OpenStack Helm
+  use_external_ingress_controller: true
+  api:
+    ingress:
+      annotations:
+        nginx.ingress.kubernetes.io/rewrite-target: /
+        # set our default issuer
+        cert-manager.io/cluster-issuer: understack-cluster-issuer
+    external_policy_local: false
+    node_port:
+      enabled: false
+
+conf:
+  octavia:
+    api_settings:
+      enabled_provider_drivers: >-
+        ovn: "The Octavia OVN driver",
+        amphora: "The Octavia Amphora driver"
```

Comment:
Any reason to enable amphora? We don't have VMs so these would be pretty heavy haproxy nodes.